crates/host/src/wasmbus/mod.rs

```diff
@@ -20,15 +20,17 @@ use std::task::{Context, Poll};
 use std::time::Duration;
 
 use anyhow::{anyhow, bail, ensure, Context as _};
-use async_nats::jetstream::kv::{Entry as KvEntry, Operation, Store};
+use async_nats::jetstream::kv::Store;
 use base64::engine::general_purpose::STANDARD;
 use base64::Engine;
 use bytes::{BufMut, Bytes, BytesMut};
+use claims::{Claims, StoredClaims};
 use cloudevents::{EventBuilder, EventBuilderV10};
 use futures::future::Either;
 use futures::stream::{AbortHandle, Abortable, SelectAll};
 use futures::{join, stream, try_join, Stream, StreamExt, TryFutureExt, TryStreamExt};
 use hyper_util::rt::{TokioExecutor, TokioIo};
+use jetstream::{create_bucket, ComponentSpecification};
```

Comment:
To ensure complete backwards compat, we need to have a `pub use jetstream::ComponentSpecification` since it was `pub` at this top level before

Reply:
Great catch thank you!